drone-kubernetes-helm/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mandrean/drone-kubernetes-helm:v0.3.0-k8s-v1.6-4-helm-v2.4.2
```

Comment:
Is there any risk this dependency changes or disappear? If so, can we avoid this dependency?

Reply:
Yes, maybe we can fork it. Could disappear, I believe